packages/modules/fsm/src/pages/employee/EditApplication/index.js

```diff
@@ -547,34 +548,34 @@ const ModifyApplication = ({ parentUrl, heading = "Modify Application" }) => {
           },
           disable: true,
         },
-        {
-          label: t("ES_EDIT_APPLICATION_ADDITIONAL_TRIP"),
-          type: "text",
-          populators: {
-            name: "additionalTrip",
-          },
-        },
-        {
-          label: t("ES_EDIT_APPLICATION_ADDITIONAL_CHARGES"),
-          type: "text",
-          populators: {
-            name: "additionalCharges",
-          },
-        },
-        {
-          label: t("ES_EDIT_APPLICATION_REASON_FOR_ADDITIONAL_CHARGES"),
-          type: "text",
-          populators: {
-            name: "reasonForAdditionalCharges",
-          },
-        },
-        {
-          label: t("ES_EDIT_APPLICATION_AMOUNT_TO_BE_PAID"),
-          type: "text",
-          populators: {
-            name: "additionalAmount",
-          },
-        },
```

Comment:
why you remove the above 4 fields?
@saurabh-egov told me that we should render them

Reply:
we can hide them for now